### ISSUE TITLE
Stopped working on 13.0.1 RC1

### ISSUE THREAD

**Issue by oparoz** [nextcloud/files_pdfviewer#61]
### Steps to reproduce
1. Open a PDF file from the Web interface

### Expected behaviour

Show PDF

### Actual behaviour

No PDF

### Server configuration

c.nc.c

### Client configuration
**Browser:** Chromium

**Operating system:** Win10

NC language set to German

#### Browser log
```
Refused to execute script from '/files_pdfviewer/vendor/pdfjs/web/compatibility.js?v=1.2.0' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
Refused to execute script from '/files_pdfviewer/vendor/pdfjs/web/l10n.js?v=1.2.0' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
...
viewer.js?v=1.2.0:7270 Uncaught (in promise) TypeError: Cannot read property 'setLanguage' of undefined
    at webViewerInitialized (viewer.js?v=1.2.0:7270)
    at <anonymous>
```

**Comment by MorrisJobke** [comment]
It's on a 13.0.1.

@danxuliu Any idea?

@oparoz Does it happen in other browsers as well?

**Comment by oparoz** [comment]
Just tried with Vivaldi and Firefox and no problem at all. It used to work just fine on Iridium, so maybe some internal components have crashed...
Let's close this if it works for you on Chrome?

**Comment by danxuliu** [comment]
It is a cache issue; it is trying to use [_vendor/pdfjs/web/viewer.js_ from v1.4.20](https://github.com/nextcloud/files_pdfviewer/blob/50e8ce74af394f7a21827fb4f0500e2f7aac49b6/vendor/pdfjs/web/viewer.js#L7270).

**Comment by MorrisJobke** [comment]
> It is a cache issue; it is trying to use vendor/pdfjs/web/viewer.js from PDF.js v1.4.20.

But this should not happen after an upgrade, because the hashes are usually changed. Right, @juliushaertl @skjnldsv ?

**Comment by skjnldsv** [comment]
@MorrisJobke for css yes... :'(

**Comment by MorrisJobke** [comment]
> We should have bumped the version number...

Yes ... :/

**Comment by koshia** [comment]
I've had this issue on 13.0.0, current PDF viewer version 1.2.0 under Apps.  I just upgraded to 13.0.1 and the problem persisted.  Cleared out FF59/Chrome64 caches/history, still same.  Just a blank page.  I read on another issue that PDFViewer 1.2.3 fixes it?  I'm not sure if 1.2.3 is a manual update because I don't see any update notification for apps.

**Comment by skjnldsv** [comment]
https://help.nextcloud.com/t/nextcloud-13-0-1-12-0-6-and-11-0-8-available-with-improved-password-protection/28970/7

**Comment by koshia** [comment]
Just watched to post this as my problem has been resolved with the PDFViewer not working.  Looks like the viewer not working is the cause of the X-Frame-Options SAMEORIGIN error that was actively displayed under my settings area.  I tracked it down and it looks like in my /etc/nginx/snippets/ssl_params.conf there was an entry for X-Frame-Options DENY, commented it out and the PDF Viewer worked fine.

REF: https://help.nextcloud.com/t/x-frame-options-sameorigin-nc-on-nginx-keeps-warning-me/1553/10
